src/pages/layer-2.js

```diff
@@ -0,0 +1,618 @@
+          decentralization.
+        </p>
+        <p>
+          Ethereum has reached the networks capacity with{" "}
+          <Link to="https://etherscan.io/chart/tx">
+            1+ million transactions per day
+          </Link>
+          , with high demand for each of these transactions. Due to the success
+          of Ethereum and the demand to use it, gas prices have risen
+          substantially, increasing the{" "}
+          <Link to="/developers/docs/scaling/">need for scaling solutions</Link>
+          . The main goal of scalability is to increase transaction speed
+          (faster finality), and transaction throughput (high transactions per
+          second), without sacrificing decentralization or security (more on the
+          Ethereum vision).
+        </p>
+        <p>
+          The Ethereum community has taken a strong stance that it would not
+          throw out decentralization or security in order to scale. Until
+          sharding, Ethereum Mainnet (layer 1) will only be able to process
+          roughly 15 transactions per second. When demand to use Ethereum is
+          high this causes network congestion, increasing transaction fees, and
+          pricing out those who cannot afford it from using Ethereum until the
+          fees reduce. That is where layer 2 comes in.
+        </p>
+      </Content>
+
+      <Content>
+        <InfoGrid>
+          {layer2Cards.map(({ image, title, preview, body, alt }, idx) => (
+            <ExpandableCard
+              contentPreview={preview}
+              title={title}
+              svg={image}
+              alt={alt}
+              key={idx}
+            >
+              {body.map((item, idx) => (
+                <p key={idx}>{item}</p>
+              ))}
+            </ExpandableCard>
+          ))}
+        </InfoGrid>
+      </Content>
+
+      <Content>
+        <h2>Layer 2</h2>
+        <p>
+          Layer 2 refers to building other Ethereum compatible networks on top
+          of layer 1, and requires no changes to the layer 1 protocol. This lets
+          layer 1 handle security, data availability, and decentralization,
+          whilst everything on the layer above (layer 2) can handle scaling.
+          Layer 2s takes the transactional burden away from the Layer 1, and
+          posts finalized proofs back to the Layer 1 to finalize the state. By
+          taking this load away from Layer 1, the base layer will become less
+          congested, and everything becomes more scalable.
+        </p>
+        <h3>Rollups</h3>
+        <p>
+          Rollups are currently the preferred layer 2 solution for scaling
+          Ethereum. By using rollups, users can{" "}
+          <Link to="https://l2fees.info/">reduce gas fees by up to 100x</Link>{" "}
+          when comparison to a layer 1 transaction.
+        </p>
+        <p>
+          Rollups bundle (or ’roll-up’) hundreds of transactions into a single
```

Comment:
```suggestion
          Rollups bundle (or ’roll up’) hundreds of transactions into a single
```
Don't think hyphen is needed. We want verb not noun, ya?